dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentityfilter/hibernate/TrackedEntityInstanceFilter.hbm.xml

```diff
@@ -41,6 +41,9 @@
     <property name="eventFilters" type="jlbEventFilter" />
 
     <property name="translations" type="jblTranslations"/>
+
+    <!-- Sharing -->
```

Comment:
I would remove the comment as it seems redundant to me. The property is already called sharing.